zerver/models/users.py

```diff
@@ -72,6 +72,7 @@ class UserBaseSettings(models.Model):
     display_emoji_reaction_users = models.BooleanField(default=True)
     twenty_four_hour_time = models.BooleanField(default=False)
     starred_message_counts = models.BooleanField(default=True)
+    receives_typing_notifications = models.BooleanField(default=True)
```

Comment:
Moved this to be next to the other typing settings in this file for better readability.